deploy/travis_deploy.sh

```diff
@@ -12,12 +12,21 @@ set -e
 CF_PATH="."
 if [ "$TRAVIS_BRANCH" == "master" ]
 then
-	CF_MANIFEST="manifests/manifest-master.yml"
-	CF_SPACE="deck-prod"
+  CF_MANIFEST="manifests/manifest-prod.yml"
+  CF_SPACE="deck-prod"
+  # Skip out of build if no tag
+  if ! [[ "$TRAVIS_TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-[A-Za-z0-9-]+)? ]]; then
```

Comment:
whats the format of the tag? is this regex looking for semver? if so, put a comment clarifying.